pkg/thing/interactors/update_data.go

```diff
@@ -73,3 +73,20 @@ func validateSchema(data entities.Data, schema []entities.Schema) bool {
 
 	return false
 }
+
+func validateSchemaNumber(value float64, valueType int) bool {
+	switch valueType {
+	case 1: /* int */
```

Comment:
considering it's a single line comment:

```suggestion
	case 1: // int
```

Reply:
same about the comments below.

Reply:
Done.